packages/core/package.json

```diff
@@ -60,6 +60,10 @@
     "./core.css": {
       "import": "./core.css",
       "require": "./core.css"
+    },
+    "./dist/core/core.css": {
+      "import": "./dist/core/core.css",
+      "require": "./dist/core/core.css"
```

Comment:
question: why we are importing it twice?

Reply:
I think better just fix the places that are importing `@juntossomosmais/dist/core/core.css` to import `@juntossomosmais/core/core.css` too, I remember that our storybook apps inside project are importing as `dist/core/core.css`, right?